pandas/tests/indexes/period/test_arithmetic.py

```diff
@@ -11,7 +11,63 @@
 import pandas.core.indexes.period as period
 
 
+_common_mismatch = [pd.offsets.YearBegin(2),
+                    pd.offsets.MonthBegin(1),
+                    pd.offsets.Minute()]
+
+
+@pytest.fixture(params=[timedelta(minutes=30),
+                        np.timedelta64(30, 's'),
+                        Timedelta(seconds=30)] + _common_mismatch)
+def not_hourly(request):
```

Comment:
Could I bother you for docstrings on these? They're printed out when running `pytest --fixtures`, which I find really helpful in understanding what fixtures we already provide.

Reply:
Just pushed a commit with these added.  Let me know if they need to be fleshed out etc.

If you find the motivation, I've got a handful of small PRs that need a nudge to get over the line (i.e. haven't gotten the "ping on green", but plausibly could): #19582, #19650, #19692, #19365, #19611, #19661.  Less trivial but important from a blocker-for-bugfixes standpoint is #19672.

Reply:
woa didn't realize fixtures prints these out. let' fix these next for all ones that don't have them. il'll create an issue

Reply:
https://github.com/pandas-dev/pandas/issues/19731